### PR TITLE
add check in ph5_validate for array table [location/Z/units_s] =unknown

### PR DESCRIPTION
**Pull request by timronan**
Thank your for contributing to PH5

Before submitting a PR, please review the pull request guidelines:
https://github.com/PIC-IRIS/PH5/blob/master/CONTRIBUTING.md#submitting-a-pull-request

### What does this PR do?
Adds a check in PH5_Validate in for location/Z/units_s=Unknown. 

### Relevant Issues?
Issue #462 

### Checklist
- [ ] This PR is not directly related to an existing issue (which has no PR yet).
- [ ] All tests pass.
- [ ] Any new or changed features have are documented.
- [ ] Changes have been added to `CHANGELOG.txt` .
- [ ] First time contributors have added your name to `CONTRIBUTORS.txt` .

### CODE REVIEW

**Review by hrotman-pic — APPROVED** [review]
Primary test archive: copy of 20-001 I rebuilt to test server upgrades.

Validate output for all channels seems to include:
`WARNING: location/Z/units_s is set unknown. Consider updating this unit to m.`

I tested on node PH5s that are archived, but we still have a copy on our servers, and validate also issues a warning for this.

Approved: I think this works as expected.  Hopefully unknown Z units will continue to not cause any problems, but now we can identify it if it does cause a problem.

File: ph5/utilities/ph5validate.py [hunk 1]
```diff
@@ -436,6 +437,11 @@ def check_station_completeness(self, station):
         e, w = validation.check_lat_lon_elev(station)
         error = error + e
         warning = warning + w
+        if station['location/Z/units_s'] in ['unknown',
+                                             'UNKNOWN']:
+            warning.append("location/Z/units_s is set " +
+                           "unknown. Consider updating " +
+                           "this unit to m.")
```
Comment (damhuonglan) [inline ph5/utilities/ph5validate.py:444]: I think these lines should be moved in under validation.py.check_lat_lon_elev().
To include all combinations of 'unknown', the condition should be changed to:
`if station['location/Z/units_s'].lower() == 'unknown':`

**Review by damhuonglan — APPROVED** [review]
I have some recommendation. You can change it before you merge the PR on your own.

**Review by gbbofh — APPROVED** [review]

File: ph5/utilities/validation.py [hunk 2]
```diff
@@ -366,7 +366,9 @@ def check_lat_lon_elev(station):
                       % station['location/Y/value_d'])
     if station['location/Y/units_s'] in [None, '']:
         warnings.append("No Station location/Y/units_s value found.")
-
+    if station['location/Z/units_s'] in ['unknown', 'UNKNOWN', 'Unknown']:
```
Comment (damhuonglan) [inline ph5/utilities/validation.py:369]: To cover all combination of unknown, I think this line should be written like this:
`if station['location/Z/units_s'].lower() == 'unknown':`

**Pull request merged**